layers/+lang/scala/layers.el

```diff
@@ -9,7 +9,7 @@
 ;;
 ;;; License: GPLv3
 
-(configuration-layer/declare-layer-dependencies '(java))
+(configuration-layer/declare-layer-dependencies '(java lsp))
```

Comment:
This is not necessary, `dap` layer already includes lsp

Reply:
It was necessary for me. I did not get either `dap` or `lsp` keybindings in my major mode map for Scala until I added this.

Reply:
Strange, anyway requiring it here too does not do any harm, so lets keep it.